### PR TITLE
Admins should be able to submit a sitemap of articles

### PR DESCRIPTION
**Pull request by samakshjain**
[#126384857]

### CODE REVIEW

File: shop/cms/views.py [hunk 1]
```diff
@@ -43,7 +44,16 @@ def sitemap_index():
     """
     Returns a Sitemap Index Page
     """
-    return __doc__
+    articles =  Article.query.all()
```
Comment (tarunbhardwaj) [inline shop/cms/views.py]: only published articles

**Comment by samakshjain** [comment]
@tarunbhardwaj Fixed.

File: shop/cms/views.py [hunk 2]
```diff
@@ -43,7 +44,18 @@ def sitemap_index():
     """
     Returns a Sitemap Index Page
     """
-    return __doc__
+    articles =  Article.query.filter_by_domain(
+        [('state', '=', 'published')]
+    ).all()
+    pages = []
+    for article in articles:
+        pages.append([article.get_absolute_url(), article.write_date])
```
Comment (tarunbhardwaj) [inline shop/cms/views.py]: article.write_date or article.create_date

Comment (samakshjain) [inline shop/cms/views.py]: @tarunbhardwaj There is a column called `published_date`, is that better than using write_date or `create_date`?

Comment (tarunbhardwaj) [inline shop/cms/views.py]: yes use that

**Pull request merged**